.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Codespell Development Environment",
+    // TODO: Switch to image 'mcr.microsoft.com/devcontainers/python:1-3' once
+    // https://github.com/codespell-project/codespell/issues/2897 has been
+    // resolved.
+    "image": "mcr.microsoft.com/devcontainers/python:1-3-bullseye",
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "github.vscode-github-actions",
+                "ms-python.python",
+                "python.vscode-pylance"
```

Comment:
According to https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance, this should be `ms-python.vscode-pylance`.

Reply:
Appreciate you pointing that out. I was apparently using the old name which I pulled from a previous config. Both still resolve correctly, but Microsoft has been in the process of renaming these for a while. Will update this.